go.mod

```diff
@@ -14,8 +14,9 @@ require (
 	github.com/onsi/ginkgo/v2 v2.17.1
 	github.com/onsi/gomega v1.32.0
 	github.com/pkg/errors v0.9.1
+	github.com/spf13/pflag v1.0.5
 	github.com/vmware/cloud-provider-for-cloud-director v0.0.0-20240725204714-a0a0e916a5ed
-	github.com/vmware/go-vcloud-director/v2 v2.21.0
+	github.com/vmware/go-vcloud-director/v2 v2.26.0-alpha.6
```

Comment:
v2.26.0 is already released right? can't we use a released version instead of an alpha version?

Reply:
I second this. We can use the 2.26.0 version that was released fairly recently.

Reply:
updated to release version. Thanks for bringing this up.